features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/SessionReplayRecorder.kt

```diff
@@ -74,6 +76,7 @@ internal class SessionReplayRecorder : OnWindowRefreshedCallback, Recorder {
         resourcesWriter: ResourcesWriter,
         rumContextProvider: RumContextProvider,
         privacy: SessionReplayPrivacy,
+        imagePrivacy: ImagePrivacy,
```

Comment:
I think step by step this constructor is blowing up, maybe we should consider using something else here (a builder maybe ?)

Reply:
I've opened a [ticket](https://datadoghq.atlassian.net/browse/RUM-5446) to do this refactor in a separate PR